src/applications/mhv/medical-records/tests/components/ChemHemDetails.unit.spec.jsx

```diff
@@ -1,83 +1,65 @@
 import React from 'react';
 import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import { beforeEach } from 'mocha';
 import reducer from '../../reducers';
 import ChemHemDetails from '../../components/LabsAndTests/ChemHemDetails';
+import chemHem from '../fixtures/chemHem.json';
+import { convertLabsAndTestsRecord } from '../../reducers/labsAndTests';
 
 describe('Chem Hem details component', () => {
-  const mockChemHem = {
-    name: 'Complete blood count',
-    category: 'Chemistry and hematology',
-    sampleTested: 'Blood',
-    orderedBy: 'Beth M. Smith',
-    requestedBy: 'John J. Lydon',
-    id: 121,
-    date: '2022-06-14T17:42:46.000Z',
-    orderingLocation:
-      '01 DAYTON, OH VAMC 4100 W. THIRD STREET , DAYTON, OH 45428',
-    collectingLocation:
-      '01 DAYTON, OH VAMC 4100 W. THIRD STREET , DAYTON, OH 45428',
-    comments: ["Jesse Roberts' blood panel is standard"],
-    results: [
-      {
-        name: 'WBC',
-        result: '5.0 K/ccm',
-        standardRange: '4.5 - 10.5 K/ccm',
-        status: 'Final',
-        labLocation:
-          '01 DAYTON, OH VAMC 4100 W. THIRD STREET , DAYTON, OH 45428',
-        interpretation: 'ref. range prior to 1/16/03 was 26-71 mg/dL.',
-      },
-    ],
-  };
-
   const initialState = {
     mr: {
       labsAndTests: {
-        labsAndTestsDetails: mockChemHem,
+        labsAndTestsDetails: convertLabsAndTestsRecord(chemHem),
       },
     },
   };
 
-  const setup = (state = initialState) => {
-    return renderWithStoreAndRouter(
-      <ChemHemDetails record={mockChemHem} fullState={state} />,
+  let screen;
+  beforeEach(() => {
+    screen = renderWithStoreAndRouter(
+      <ChemHemDetails
+        record={convertLabsAndTestsRecord(chemHem)}
+        fullState={initialState}
+      />,
       {
-        initialState: state,
+        initialState,
         reducers: reducer,
-        path: '/labs-and-tests/121',
+        path: '/labs-and-tests/ex-MHV-chReport-1',
       },
     );
-  };
+  });
 
   it('renders without errors', () => {
-    const screen = setup();
     expect(screen);
   });
 
   it('should display the test name', () => {
-    const screen = setup();
-    const header = screen.getAllByText(mockChemHem.name, {
-      exact: true,
-      selector: 'h1',
-    });
+    const header = screen.getAllByText(
+      initialState.mr.labsAndTests.labsAndTestsDetails.name,
+      {
+        exact: true,
+        selector: 'h1',
+      },
+    );
     expect(header).to.exist;
   });
 
   it('should display the test results', () => {
-    const screen = setup();
-    const results = screen.getByText(mockChemHem.results[0].name, {
-      exact: true,
-      selector: 'h3',
-    });
+    const results = screen.getByText(
+      initialState.mr.labsAndTests.labsAndTestsDetails.results[0].name,
+      {
+        exact: true,
+        selector: 'h3',
+      },
+    );
     expect(results).to.exist;
   });
```

Comment:
Why did we switch this test from checking the screen to checking the redux state? Same question for two other tests in this PR.

Reply:
I thought maybe this was a better way that perhaps we should switch to.

Reply:
Originally this test was looking at mockdata right in the unit test file. I changed it to use mockdata in a separate file that was then loaded into redux. I've now changed it again to just check using a string because I think that's a little more straightforward. What do you think?

Reply:
Oh I see -- sorry I actually mis-read your code when I made that comment. Looks good!